Docker-dev-build

```diff
@@ -0,0 +1,135 @@
+FROM debian:bullseye
+
+# # which spack version are we using now? Default is develop
+# # but other strings can be given to the docker build command
+# # (for example docker build --build-arg SPACK_VERSION=v0.16.2)
+ARG SPACK_VERSION=develop
+ARG OCT_VERSION=12.1
+RUN echo "Building with spack version ${SPACK_VERSION}"
+
+# Any extra packages to be installed in the host
+ARG EXTRA_PACKAGES
+RUN echo "Installing EXTRA_PACKAGES ${EXTRA_PACKAGES} on container host"
+
+# general environment for docker
+ENV SPACK_ROOT=/home/user/spack \
+	  SPACK=/home/user/spack/bin/spack \
+	  FORCE_UNSAFE_CONFIGURE=1
+
+RUN apt-get -y update
+# Convenience tools, if desired for debugging etc
+# RUN apt-get -y install wget time nano vim emacs git
+
+# From https://github.com/ax3l/dockerfiles/blob/master/spack/base/Dockerfile:
+# install minimal spack dependencies
+RUN apt-get install -y --no-install-recommends \
+              autoconf \
+              build-essential \
+              ca-certificates \
+              coreutils \
+              curl \
+              environment-modules \
+	            file \
+              gfortran \
+              git \
+              openssh-server \
+              python \
+              unzip \
+              vim \
+              wget \
+           && rm -rf /var/lib/apt/lists/*
+
+# load spack environment on login
+RUN echo "source $SPACK_ROOT/share/spack/setup-env.sh" \
+           > /etc/profile.d/spack.sh
+
+RUN adduser user
+USER user
+WORKDIR /home/user
+
+# install spack
+RUN git clone https://github.com/spack/spack.git
```

Comment:
add the `-c feature.manyFiles=true` . (I know we haven't done this in other places, but we probably should.)

Reply:
[7e8d09a](https://github.com/fangohr/octopus-in-spack/pull/56/commits/7e8d09aa53cfb4c9c6e05a447ac015c4cd7b9182)